app/views/users/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<!--<form accept-charset="UTF-8" action="/users" method="POST">
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+    <label id = "username" for ="username">username</label>
+    <input type="text" name="user[username]"><br>
+    <label id = "email" for ="email">email</label>
+    <input type="text" name="user[email]"><br>
+    <label id = "password" for ="password">pasword</label>
+    <input type="password" name="user[password]"><br>
+    <input type="submit" value="Submit">      
+</form>-->
+
+<!--<%= form_tag("/users", method: "post") do %>
```

Comment:
Please refer line 1 of this file for setting your URL to your forms

Reply:
We at the momento of making the forms work figured out that this is the way the forms are working for us, we fixed the rest of the issues, can you explain more in depth how to and how is it useful doing it this way? thanks